.clang-format

```
@@ -0,0 +1,11 @@
+---
+BasedOnStyle: Google
+ColumnLimit: '100'
+IndentPPDirectives: None
+IndentWidth: '4'
+MaxEmptyLinesToKeep: '1'
+SpacesInParentheses: 'false'
+TabWidth: '4'
```

```suggestion
IndentWidth: '3'
MaxEmptyLinesToKeep: '1'
SpacesInParentheses: 'false'
TabWidth: '3'
```
is the standard in the ROOT world: https://github.com/root-project/root/blob/master/.clang-format